METRICS.md

```diff
@@ -3,26 +3,26 @@
-/intel/yarn/queue/*/capacity | Configured queue capacity in percentage relative to its parent queue
-/intel/yarn/queue/*/maxactiveapplications | The maximum number of active applications this queue can have
-/intel/yarn/queue/*/maxactiveapplicationsperuser | The maximum number of active applications this queue can have
-/intel/yarn/queue/*/maxapplications | The maximum number of active applications this queue can have
-/intel/yarn/queue/*/maxapplicationsperuser | The maximum number of active applications per user this queue can have
-/intel/yarn/queue/*/maxcapacity | Configured maximum queue capacity in percentage relative to its parent queue
-/intel/yarn/queue/*/numactiveapplications | The number of pending applications for this user in this queue
-/intel/yarn/queue/*/numapplications | The number of applications currently in the queue
-/intel/yarn/queue/*/numcontainers | The number of containers being used
-/intel/yarn/queue/*/numpendingapplications | The number of pending applications for this user in this queue
-/intel/yarn/queue/*/usedcapacity | Used queue capacity in percentage
-/intel/yarn/queue/*/usedresources | Used queue resources
-/intel/yarn/queue/*/userlimit | The minimum user limit percent set in the configuration
-/intel/yarn/queue/*/userlimitfactor | The user limit factor set in the configuration
-/intel/yarn/queue/*/resources_memory | The amount of memory used (in MB)
-/intel/yarn/queue/*/resources_vcores | The number of virtual cores
-/intel/yarn/schedulerinfo/capacity | Configured queue capacity in percentage for root queue
-/intel/yarn/schedulerinfo/maxcapacity | Configured maximum queue capacity in percentage for root queue
+Namespace | Data Type | Description
+----------|-----------|-------------
+/intel/yarn/queue/[queue_id]/absolutecapacity | float64 | Absolute capacity percentage this queue can use of entire cluster
+/intel/yarn/queue/[queue_id]/absolutemaxcapacity | int | Absolute maximum capacity percentage this queue can use of the entire cluster
+/intel/yarn/queue/[queue_id]/absoluteusedcapacity | int | Absolute used capacity percentage this queue is using of the entire cluster
+/intel/yarn/queue/[queue_id]/capacity | int | Configured queue capacity in percentage relative to its parent queue
+/intel/yarn/queue/[queue_id]/maxactiveapplications | int | The maximum number of active applications this queue can have
+/intel/yarn/queue/[queue_id]/maxactiveapplicationsperuser | int | The maximum number of active applications this queue can have
+/intel/yarn/queue/[queue_id]/maxapplications | int | The maximum number of active applications this queue can have
+/intel/yarn/queue/[queue_id]/maxapplicationsperuser | int | The maximum number of active applications per user this queue can have
+/intel/yarn/queue/[queue_id]/maxcapacity | int | Configured maximum queue capacity in percentage relative to its parent queue
```

Comment:
Are you sure this is int? Please, look on code or even o this example of response - https://github.com/intelsdi-x/snap-plugin-collector-yarn/blob/master/yarncollector/response.json#L102

Reply:
@kdembler, I could you check if declared data types for each of listed metrics is correct?

Reply:
https://github.com/intelsdi-x/snap-plugin-collector-yarn/blob/master/yarn/xml.go#L57 Apparently even though JSON response contains data that looks like `float`, plugin parses it to `int`, and this is the type that is exposed.

Reply:
ok, so then it looks good to me